src/locales/en.locale.json

```diff
@@ -519,6 +521,9 @@
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_body": "Do you really want to delete your comment?",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_cancel": "Cancel",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_delete": "Delete",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment.hide": "Hide Comments",
```

Comment:
Rename this key to `mno_enterprise.templates.dashboard.marketplace.show.hide_comments`. If a key is used (ie. `mno_enterprise.templates.dashboard.marketplace.show.comment`) we cannot reuse it by adding `.hide`. It has to be translated to YAML and this format do not accept it.